selftests/functional/test_utils_podman.py

```diff
@@ -0,0 +1,19 @@
+import asyncio
+
+from avocado import Test
+from avocado.utils.podman import Podman
+
+
+class PodmanTest(Test):
+
+    def test_python_version(self):
+        """
+        :avocado: requirement={"type": "package", "name": "podman", "action": "check"}
+        :avocado: tags=slow
+        """
+        podman = Podman()
+        loop = asyncio.get_event_loop()
+        coro = podman.get_python_version('fedora:34')
+        result = loop.run_until_complete(coro)
+        self.assertTrue(type(result), tuple)
+        self.assertTrue(len(result), 3)
```

Comment:
Ditto.  But here you can compare against the actual Python version, because `fedora:34` will always have the same one.  And while at it, 35 is out, so maybe that's even better.